src/rvi.c

```diff
@@ -1050,6 +1050,38 @@ TRviHandle rviInit ( char *configFilename )
     return NULL;
 }
 
+
+TRviHandle rviInit (char *configFilename)
```

Comment:
No, there wasn't const, but I think it should be. And I don't  know the reason why it is not const.